cmd/root.go

```diff
@@ -50,8 +54,22 @@ HPC deployments on the Google Cloud Platform.`,
 // Execute the root command
 func Execute() error {
 	if len(GitCommitInfo) > 0 {
+		dir, err := os.Getwd()
```

Comment:
nit: this is assuming your are working from the same location as ghpc. But what if ghpc is in path and the user is somewhere else? Could we also check where the binary is located? (also difficult to figure out what is best).

Reply:
also, what if you are not in a hpc-toolkit repo, but in another git repo?

Reply:
**To the first comment:**, my thinking for using workdir instead of binary site was that if the user had an official release installed under `/usr/bin` or `~/bin` or in another directory added to their `PATH` AND was developing in a working directory with ghpc source code, it would still warn the user in case they call `$ ghpc` instead of `$ ./ghpc`.

**To the second comment: **, we could also  compare the initial commits of the working directory vs binary using something similar to `git rev-list --max-parents=0 HEAD` which would prevent false negatives in situations where the user has a working directory with a git branch that is a few commits behind their binary.

Reply:
I updated the logic as follows. Code will traverse the working directory tree until it hits the root of the file system, looking for a hpc-toolkit repo. If repo not found, we will then look at the hash of the call site.

`hpc-toolkit` repo is positively identified by searching for the initial commit (injected at build time) in the current repository.